graphdatascience/procedure_surface/arrow/wcc_arrow_endpoints.py

```diff
@@ -0,0 +1,236 @@
+from typing import Any, List, Optional
+
+from pandas import DataFrame
+
+from ...arrow_client.authenticated_arrow_client import AuthenticatedArrowClient
+from ...arrow_client.v2.job_client import JobClient
+from ...arrow_client.v2.mutation_client import MutationClient
+from ...arrow_client.v2.write_back_client import WriteBackClient
+from ...graph.graph_object import Graph
+from ..api.wcc_endpoints import WccEndpoints, WccMutateResult, WccStatsResult, WccWriteResult
+
+WCC_ENDPOINT = "v2/community.wcc"
+
+
+class WccArrowEndpoints(WccEndpoints):
+    def __init__(self, arrow_client: AuthenticatedArrowClient, write_back_client: Optional[WriteBackClient]):
+        self._arrow_client = arrow_client
+        self._write_back_client = write_back_client
+
+    def mutate(
+        self,
+        G: Graph,
+        mutate_property: str,
+        threshold: Optional[float] = None,
+        relationship_types: Optional[List[str]] = None,
+        node_labels: Optional[List[str]] = None,
+        sudo: Optional[bool] = None,
+        log_progress: Optional[bool] = None,
+        username: Optional[str] = None,
+        concurrency: Optional[int] = None,
+        job_id: Optional[str] = None,
+        seed_property: Optional[str] = None,
+        consecutive_ids: Optional[bool] = None,
+        relationship_weight_property: Optional[str] = None,
+    ) -> WccMutateResult:
+        config = self._build_configuration(
+            G,
+            concurrency,
+            consecutive_ids,
+            job_id,
+            log_progress,
+            None,
+            node_labels,
+            relationship_types,
+            relationship_weight_property,
+            seed_property,
+            sudo,
+            threshold,
+        )
+
+        job_id = JobClient.run_job_and_wait(self._arrow_client, WCC_ENDPOINT, config)
+
+        mutate_result = MutationClient.mutate_node_property(self._arrow_client, job_id, mutate_property)
+        computation_result = JobClient.get_summary(self._arrow_client, job_id)
+
+        return WccMutateResult(
+            computation_result["componentCount"],
+            computation_result["componentDistribution"],
+            computation_result["preProcessingMillis"],
+            computation_result["computeMillis"],
+            computation_result["postProcessingMillis"],
+            0,
```

Comment:
TODO: measure the actual time

Reply:
Are we not measuring this already server-side?